### ISSUE TITLE
Dysembryoplastic neuroepithelial tumor

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6734]
Preferred term label:
Dysembryoplastic neuroepithelial tumor

Synonyms:
DNET, DNT

Definition (free text, please give PubMed ID)
Dysembryoplastic neuroepithelial tumors are rare tumours of the central nervous system.

Comment: They are commonly found in the temporal lobe and may be associated with temporal lobe epilepsy. DNET's are sometimes found in association with focal cortical dysplasia and have both neuronal and glial features.

PMID: 25408925

Parent term (use hpo.jax.org/app)
Neuronal/glioneuronal neoplasm of the central nervous system HP:0025170

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
Temporal lobe epilepsy

Your nano-attribution (ORCID)
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
added
